libraries/AP_HAL_SITL/GPIO.cpp

```diff
@@ -63,7 +64,7 @@ bool GPIO::attach_interrupt(uint8_t interrupt_num, AP_HAL::Proc p,
     return true;
 }
 
-bool GPIO::usb_connected(void)
+bool GPIO::usb_connected()
```

Comment:
I'd strongly rather keep the `(void)` expression on function definitions and declaration, and I believe this is the consistent widespread way the code base is written.

Reply:
Ok I can keep it, even if I don't find it usefull. And it is more or less consistent in the code base ^^

Reply:
I actually don't like the void, it's completely useless. Also, it's certainly not consistent across the code base. That said, if @WickedShell feels strongly about it, I'd say to leave it.